### PR TITLE
feat(cxl-marketing-nav): show search declaratively

### PR DESCRIPTION
**Pull request by tomasced**
Related issue: https://github.com/conversionxl/jobs-theme/issues/132

### CODE REVIEW

File: packages/cxl-ui/src/components/cxl-marketing-nav.js [hunk 1]
```diff
@@ -35,6 +35,10 @@ export class CXLMarketingNavElement extends LitElement {
   @property({ type: Boolean, reflect: true })
   minimal = false;
 
+  // Conditionally hide search box
+  @property({ type: Boolean, reflect: true })
+  hideSearch = false;
```
Comment (lkraav) [inline packages/cxl-ui/src/components/cxl-marketing-nav.js:40]: Configurations towards the negative are mentally taxing.

Refactor to `showSearch` instead.

File: packages/cxl-ui/src/components/cxl-marketing-nav.js [hunk 2]
```diff
@@ -317,6 +318,10 @@ export class CXLMarketingNavElement extends LitElement {
       searchElement = this.querySelector('.menu-item-search');
     }
 
+    if (this.hideSearch) {
+      searchElement.setAttribute('hidden', true);
+    }
```
Comment (lkraav) [inline packages/cxl-ui/src/components/cxl-marketing-nav.js:323]: Can we maybe skip whole part of template instead of build-hide?

**Review by lkraav — CHANGES_REQUESTED** [review]

File: packages/cxl-ui/src/components/cxl-marketing-nav.js [hunk 3]
```diff
@@ -110,6 +111,13 @@ export class CXLMarketingNavElement extends LitElement {
     `;
   }
 
+  /**
+   * Hide search when minimal view is enabled
+   */
+  isSearchHidden() {
```
Comment (lkraav) [inline packages/cxl-ui/src/components/cxl-marketing-nav.js:117]: Avoid creating "used once" new symbols.

Can we write what this method does directly into template `?hidden` condition?

**Review by lkraav — CHANGES_REQUESTED** [review]

**Comment by pawelkmpt** [comment]
Obsolete

**Pull request closed without merging**